internal/olm/client/client.go

```diff
@@ -211,14 +209,14 @@ func (c Client) DoCSVWait(ctx context.Context, key types.NamespacedName) error {
 	return wait.PollImmediateUntil(time.Second, csvPhaseSucceeded, ctx.Done())
 }
 
-func (c Client) GetInstalledVersion(ctx context.Context) (string, error) {
-	opts := client.InNamespace(OLMNamespace)
+func (c Client) GetInstalledVersion(ctx context.Context, namespace string) (string, error) {
+	opts := client.InNamespace(namespace)
```

Comment:
Is not this value in the c(Client) now? Could not it be `client.InNamespace(c.Namespace)`?